src/ngAria/aria.js

```diff
@@ -83,7 +83,8 @@ function $AriaProvider() {
     ariaMultiline: true,
     ariaValue: true,
     tabindex: true,
-    bindKeypress: true
+    bindKeypress: true,
+    keypressNodeNames: ['DIV', 'LI']
```

Comment:
configurable is good, but I think we shouldn't expose an array for it at least.

If we decide we don't want it to be configurable and just "do the right thing always" later, it's trivial to replace configuration methods with `noop`, and not have anything unnecessary taking up space in the framework. So I see that as being preferable, tbh